test/e2e/multicast_test.go

```diff
@@ -56,36 +61,64 @@ func TestMulticast(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error computing multicast interfaces: %v", err)
 	}
+	externalHostIface, err = getMulticastExternalHostIface(externalHostIdx)
+	if err != nil {
+		t.Fatalf("Error getting external host interface: %v", err)
+	}
+	t.Run("testMulticastWithNoEncap", func(t *testing.T) {
+		runMulticastTestCases(t, data, nodeMulticastInterfaces, true)
+	})
+	t.Run("testMulticastWithEncap", func(t *testing.T) {
```

Comment:
This is copied from https://github.com/antrea-io/antrea/pull/3947? Maybe remove it to keep this patch clean?

Reply:
updated